src/pkg/packaging/dir.proj

```diff
@@ -164,7 +164,11 @@
 
   <Target Name="GenerateCombinedInstallers" DependsOnTargets="InitPackage;GenerateBundles" />
 
-  <Target Name="GenerateNugetPackages" DependsOnTargets="InitPackage" Condition="'$(UsePrebuiltPortableBinariesForInstallers)' == 'false'">
+  <PropertyGroup>
+    <PackageManagedComponents>false</PackageManagedComponents>
+  </PropertyGroup>
+
+  <Target Name="GenerateNugetPackages" DependsOnTargets="InitPackage" Condition="'$(UsePrebuiltPortableBinariesForInstallers)' == 'false' and '$(PackageManagedComponents)' == 'true'">
```

Comment:
Instead of going this route can you instead put a condition on the ItemGroup for the PackageProjects below similar to what we are going to do in corefx and only include them if the BuildAllPackages property is true.